docs/.gitignore

```diff
@@ -0,0 +1,5 @@
+_site/
+deploy/
+node_modules/
+scripts/
+package-lock.json
```

Comment:
Should these be here?  As far as I can tell only `_site/` and `_deploy/` are moving under `docs/`, but not the scripts and node environment?

Reply:
These were pre-existed but since there isn't any need for them I have removed them here https://github.com/OAI/OpenAPI-Specification/pull/3868/commits/83770b9c94428b6320070646bb6d5c55da7e60df